### DISCUSSION TITLE
Use Project code in Purchase Order Report

### DISCUSSION THREAD

**Discussion by kevplex911** [inventree/InvenTree#7058] [Q&A]
Hey guys 

Anyone know if i call the project code assigned to purchase order in reports ?
I search in docs but found nothing 

Thank you

**Comment by SchrodingersGat** [comment]
@kevplex911 yep this is possible. I made a quick example using the [demo server](https://demo.inventree.org)

![image](https://github.com/inventree/InvenTree/assets/10080325/6f999b0a-8796-450e-825d-5d890c8fda3a)

**Reply by kevplex911** [reply]
Nice exactly what I need thanks a lot
